packages/fxa-settings/tailwind.config.js

```diff
@@ -0,0 +1,181 @@
+module.exports = {
+  purge: {
+    content: ['./src/**/*.tsx', './public/index.html'],
+    whitelist: [],
+  },
+  theme: {
+    screens: {
+      sm: '480px',
```

Comment:
I think we may actually want to call these `mobileLandscape`, `tablet`, `desktop`, and `desktopXl` to make it more clear what we're targeting. (That's my preference anyway)

Reply:
Ah, I thought I had addressed this. I will make this change.